index.html

```diff
@@ -0,0 +1,236 @@
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w2">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w3">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w4">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w5">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w6">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+    <section class="AboutMe">
+      <div class="about">
+        <h1>About me</h1>
+        <p>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
```

Comment:
- [ ] I have realized that the text inside this button does not fit well because the text size is bigger than the required one. Kindly ensure that you fix this. 
- [ ] Also, as I have requested earlier, kindly fix the button state so that it matches the provided button states as it is in the Figma design.
- [ ] Also, kindly ensure that you give it a margin/ padding-top so that there is some spacing between it an the texts that are above it.

**This is how I am seeing this button:**

![image](https://user-images.githubusercontent.com/86318284/192719749-a3171d15-a579-41e3-b8c8-6d74dcdc9edd.png)

**This is the desired button design:**

![image](https://user-images.githubusercontent.com/86318284/192719821-79b128de-52d3-4e85-8c51-49c7c5da8f4b.png)